packages/blockfrost/test/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../../../tsconfig.json",
+  "extends": "../../../test/tsconfig.json",
```

Comment:
:eyes:

Reply:
`"../../../test/tsconfig.json"` is a new one, so it's really a new configuration